scenario_editor/src/Home/Home.js

```diff
@@ -24,14 +32,16 @@ function Home() {
 
       <div class="Right-Button">
         <Button
-        size = "lg"
+        size = "custom-button-size"
         variant="outline-secondary" 
         href=
         "https://duckduckgo.com/?q=How+to+make+a+scenario&t=h_&ia=web">
           Edit an Existing Scenario
         </Button>{' '}
       </div>
     </div>
+
```

Comment:
there's a lot of weird indenting and spacing in these files. I suggest using prettier+husky dev dependencies to auto format your code and keep the coding style consistent across all devs